build.gradle

```diff
@@ -31,14 +31,17 @@ ext {
   targetCompatibilityVersion = JavaVersion.VERSION_1_7
 }
 
+def supportLibVersion = '23.1.1'
+
 ext.deps = [
   // Android
   android               : 'com.google.android:android:4.1.1.4',
   support_v4            : 'com.google.android:support-v4:r7',
-  appcompat_v7          : 'com.android.support:appcompat-v7:23.1.1',
-  support_annotations   : 'com.android.support:support-annotations:23.1.1',
+  preference_v7         : "com.android.support:preference-v7:${supportLibVersion}",
```

Comment:
it makes me think we should maybe release `lightcycles-appcompat`, `lightcycles-preferences`